tests/kernel/workq/work_queue/src/main.c

```diff
@@ -20,9 +20,14 @@
 #include <zephyr/tc_util.h>
 #include <zephyr/sys/util.h>
 
+/* Adjust timings to a slow platform. */
+#ifndef TIMEOUT_MULTIPLIER
+#define TIMEOUT_MULTIPLIER (1)
+#endif
+
 #define NUM_TEST_ITEMS          6
 /* Each work item takes 100ms */
-#define WORK_ITEM_WAIT          100
+#define WORK_ITEM_WAIT          (100 * (TIMEOUT_MULTIPLIER))
```

Comment:
Not loving the "multiplier" scheme, as it requires understanding the original rate.  Maybe something like "`CONFIG_WORK_ITEM_DELAY_MS`" with a default of 100, then set this per-platform as needed?